### ISSUE TITLE
A page listing all sites that support a certain feature.

### ISSUE THREAD

**Issue by ChildishGiant** [2factorauth/twofactorauth#3307]
I just got a u2f key and want to see what sites I use support them. A page listing only sites that support it, in one long list would be much easier to check through than going category by category.

**Comment by phallobst** [comment]
Maybe expanding the search keywords by something like "tfa:email tfa:u2f tfa:totp" etc. would be sufficient? Then a search for "tfa:u2f" would show all websites supporting U2F.

**Comment by fainpablo** [comment]
That would be awesome. Talking about 1Password, for example. They support U2F, but on the yml file is only listed "hardware". Should we replace it by "u2f" ? 

```YAML
  - name: 1Password
    url: https://1password.com/
    img: 1password.png
    tfa:
      - totp
      - hardware
    doc: https://support.1password.com/two-factor-authentication/
    exception: "Teams using Duo with 1Password can't enable additional 2FA as it's already handled by Duo"
```

**Comment by Carlgo11** [comment]
@fainpablo yes if they only support U2F then the hardware tag should be replaced. If they support U2F and some other hardware implementation then both should be present.

**Comment by phallobst** [comment]
Closed by #4584
